### PR TITLE
Update icon: nextjs (original, original-wordmark, plain, line, line-wordmark)

### PR DESCRIPTION
**Pull request by Snailedlt**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

Hopefully solves the nextjs issue here: https://github.com/devicons/devicon/pull/1998#issuecomment-1807542194

## Link to prove your SVG is correct and up-to-date.

The official versions can be found on the website: https://nextjs.org/
You can download the original versions by clicking here: 
image

## Other relevant info
The icon has changes since last time, so I updated all versions to the updated versions.
`#000` was chosen as the main color

### CODE REVIEW

**Review by canaleal — APPROVED** [review]
Great stuff. 
No artifacts in the font versions and the JSON looks good. 

image

File: icons/nextjs/nextjs-line.svg
[the remark this answers is not in the document]
Comment (Snailedlt) [reply icons/nextjs/nextjs-line.svg:1]: Agreed, that's how it looks when adding a border and removing the fill though

[the remark this answers is not in the document]
Comment (lunatic-fox) [reply icons/nextjs/nextjs-line.svg:1]: Here goes some suggestions:
### `#1 Option`


### `#2 Option`

File: icons/nextjs/nextjs-plain.svg
Comment (lunatic-fox) [inline icons/nextjs/nextjs-plain.svg:1]: Here goes more suggestions:
### `#1 Option`


### `#2 Option`

File: icons/nextjs/nextjs-line.svg
[the remark this answers is not in the document]
Comment (Snailedlt) [reply icons/nextjs/nextjs-line.svg:1]: I really like option 2, but I likely don't have time to commit it until January, so feel free to commit it whenever you have time :)

File: icons/nextjs/nextjs-plain.svg
Comment (Snailedlt) [reply icons/nextjs/nextjs-plain.svg:1]: Option 1 looks very nice imo :)

File: icons/nextjs/nextjs-plain.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M64 0A64 64 0 0 0 0 64a64 64 0 0 0 64 64 64 64 0 0 0 35.55-10.781l-52.536-67.88v40.24H38.4V38.4h10.768l32.61 42.004V38.4h8.533v51.2h-1.395l17.39 22.402A64 64 0 0 0 128 64 64 64 0 0 0 64 0zm41.8 112.438a64.216 64.216 0 0 1-2.228 1.837 64 64 0 0 0 2.229-1.838zm-3.17 2.574a64.216 64.216 0 0 1-2.638 1.896 64 64 0 0 0 2.639-1.896z"/></svg>
```
Comment (Snailedlt) [inline icons/nextjs/nextjs-plain.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M64 0A64 64 0 0 0 0 64a64 64 0 0 0 64 64 64 64 0 0 0 35.508-10.838L47.014 49.34v40.238H38.4V38.4h10.768l57.125 73.584A64 64 0 0 0 128 64 64 64 0 0 0 64 0Zm17.777 38.4h8.534v48.776L81.777 75.97Zm24.18 73.92-.111.096a64 64 0 0 0 .111-.096z"/></svg>
```

File: icons/nextjs/nextjs-line.svg [hunk 2]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M64 0C28.7 0 0 28.7 0 64s28.7 64 64 64 64-28.7 64-64S99.3 0 64 0zm32.7 114.4L48.4 41.8h-6.8v50.1h6.8V55.3l44.2 61.5c-8.5 4.6-18.2 7.2-28.6 7.2-33.2.1-60.1-26.8-60.1-60S30.8 3.9 64 3.9s60.1 26.9 60.1 60.1c0 21.1-10.9 39.7-27.4 50.4z"/><path d="M78.6 73.3l7.5 11.3V41.8h-7.5z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M64 0C28.658 0 0 28.658 0 64s28.658 64 64 64a63.683 63.683 0 0 0 35.273-10.605l.008.012.274-.185.145-.101c.114-.077.225-.158.338-.235a64.421 64.421 0 0 0 1.933-1.375c.189-.14.378-.278.565-.42a64.045 64.045 0 0 0 3.78-3.076l.25-.222-.01-.011C119.708 100.059 128 83 128 64c0-35.342-28.658-64-64-64Zm0 .707C98.96.707 127.293 29.04 127.293 64c0 18.778-8.182 35.635-21.168 47.226L89.501 89.812h1.018V38.188h-9.193V79.28L49.423 38.188H38.188V89.79h9.273V50.454l51.379 66.383A62.988 62.988 0 0 1 64 127.293C29.04 127.293.707 98.96.707 64S29.04.707 64 .707ZM38.895 38.895h10.182l32.956 42.45v-42.45h7.779v50.21h-1.756l17.53 22.581a63.512 63.512 0 0 1-6.164 4.75l-52.668-68.05v40.697h-7.859z"/></svg>
```
Comment (Snailedlt) [inline icons/nextjs/nextjs-line.svg:1]: Add line version (2)
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M64 0C28.66 0 0 28.66 0 64s28.66 64 64 64h.002a63.994 63.994 0 0 0 35.506-10.838l.014-.008.37-.283-.568-.815-52.676-68.054v40.88h-7.552V39.097h9.943l56.229 72.425.64.819.371-.344.014-.012A64.002 64.002 0 0 0 128 64c0-35.34-28.66-64-64-64Zm0 .992c34.804 0 63.008 28.204 63.008 63.008a62.999 62.999 0 0 1-20.983 46.879l-56.5-72.775H38.104v51.771h9.537v-38.97l50.863 65.712A63 63 0 0 1 64 127.007C29.196 127.008.992 98.805.992 64S29.196.992 64 .992Zm17.143 37.112v37.94l9.46 12.423V38.104Zm.992.992h7.476v46.43l-7.476-9.815z"/></svg>
```

**Review by lunatic-fox — APPROVED** [review]
Great! 👍🏼
Thank you for this contribution! 🚀
image
image
image
image

**Review by canaleal — APPROVED** [review]
Awesome work! Ready for merging.

**Pull request merged**